### PR TITLE
Code tidy up

### PR DESCRIPTION
**Pull request by vbeardow**
- Code tidy up (add missing type hints), renamed some variables
- Added README.md
- Added dependencies to poetry lock
- Fixed some type hint warnings thrown by mypy

### CODE REVIEW

File: .pre-commit-config.yaml [hunk 1]
```diff
@@ -8,4 +8,10 @@ repos:
     hooks:
       - id: pylint
         args:
-        - --fail-under=3
+        - --fail-under=8
```
Comment (Edward-Boulderstone) [inline .pre-commit-config.yaml:11]: Love watching this number go up

File: README.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+And to check the test coverage run
+
+```
+make test-cov
+```
+## Contributing
+Pull requests are welcome. For major changes, please open an issue first to discuss what you would like to change.
+
+Please make sure to update tests as appropriate.
+
```
Comment (Edward-Boulderstone) [inline README.md:39]: Nice README, might steal it

File: blackjack/actions.py [hunk 3]
```diff
@@ -35,42 +38,42 @@ def stick_or_twist(hand: Hand, deck: Deck) -> None:
...
+        elif move.lower() == "s":
             print("Player has chosen to stick.")
             game_config.playing = False
         else:
             print_invalid_input(["s", "t"])
         break
 
 
-def reveal_cards(my_hand: Hand, reveal_all=True) -> None:
+def reveal_cards(player_hand: Hand, reveal_all: Boolean = True) -> None:
```
Comment (Edward-Boulderstone) [inline blackjack/actions.py]: Why from xmlrpc.client import Boolean and not built in bool?

**Comment by Edward-Boulderstone** [comment]
Where did you get the idea of docstrings for each of your files from?

File: blackjack/actions.py (hunk 3 above)
Comment (vbeardow) [reply blackjack/actions.py]: Oops! This was an accident - nice catch.

**Comment by vbeardow** [comment]
> Where did you get the idea of docstrings for each of your files from?

Just had a look at examples from here:
https://www.programiz.com/python-programming/docstrings

**Pull request merged**